test/behavioral/behavioral.ts

```diff
@@ -31,9 +31,18 @@ export function suite(browserName: string) {
     });
 
     test("nav/guides", async testExecutionObject => {
-        await util.$x(driver, "//nav//hx-disclosure", "Guides").click();
-        let guidesText = await driver.findElement(By.id("nav-Guides")).getText();
-        testExecutionObject.deepEqual(guidesText, "Getting Started");
+        const expected = [
+            "Getting Started",
+            "FAQ",
+            "React Compatibility",
+            "Polyfills"
+        ];
+        await driver.findElement(By.css("hx-disclosure")).click();
+        await driver.findElement(By.id("nav-Guides")).click();
```

Comment:
Are lines 40 and 41 needed? 

As it's currently set up, line 40 will _expand_ the "GUIDES" section and line 41 will _collapse_ it (essentially resulting in a null op).

Reply:
await driver.findElement(By.css("hx-disclosure")).click();
This is used to locate the nav bar completely. 
await driver.findElement(By.id("nav-Guides")).click();
Will click the GUIDES section and expand it.